### DISCUSSION TITLE
http2 configured.... but become http1.1 on calls

### DISCUSSION THREAD

**Discussion by Cassetete** [rwf2/Rocket#2887] [Questions]
I've looked in the documentation and it seems that http2 is active by default. In the git examples, let's take “chat”. When I run the example, it does indeed say: “>> http2: true”. 
The problem is that in firefox, chrome and powershell, they all tell me that they've received "http1.1".

Is this normal?

**Comment by the10thWiz** [comment]
Yes, this is normal. The RFC for http/2 says TLS (1.2 or higher) is required for http/2, although the w3 working group says it's not required. Doing a quick check locally, Firefox requires TLS for http/2. I suspect that chrome and powershell also require tls for http/2.
